workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -34,7 +34,8 @@ t2g_map = ['cdna': 'Homo_sapiens.GRCh38.103.spliced.tx2gene.tsv',
 // 10X barcode files
 barcodes = ['10Xv2': '737K-august-2016.txt',
             '10Xv3': '3M-february-2018.txt',
-            '10Xv3.1': '3M-february-2018.txt']
+            '10Xv3.1': '3M-february-2018.txt',
+            '10Xv2_5prime': '737K-august-2016.txt']
```

Comment:
Doesn't matter much here, but in the final workflow, I would probably want to keep all the v2s together.